db/migrate/20170927135007_migrate_dialog_field_associations_to_use_new_relationship.rb

```diff
@@ -0,0 +1,31 @@
+class MigrateDialogFieldAssociationsToUseNewRelationship < ActiveRecord::Migration[5.0]
+  Rails.application.eager_load!
+
+  def up
+    Dialog.all.each do |dialog|
```

Comment:
Create a stub model, as is done in other data migrations, for Dialog and DialogFieldAssociation